packages/main/src/pages/protected/workspace/Workspace.jsx

```diff
@@ -87,6 +88,10 @@ export default function Index() {
           }
         }
       );
+      console.log(
+        `https://api.zuri.chat/users/${userData.email}/organizations`,
+        "omo"
+      );
```

Comment:
What was happening here?🤣. Remove this console.log too

Reply:
lol